python/python_fastapi_server/routers/edr.py

```diff
@@ -420,11 +471,16 @@ def get_params_for_collection(edr_collection: str) -> dict[str, Parameter]:
         else:
             label = param_el["name"]
 
+        symbol = Symbol(value=param_el["unit"], type=SYMBOL_TYPE_URL)
+        param_id = param_el["name"]
+        # If standard name was configured, use that instead with a vocabulary
+        if "standard_name" in param_el:
+            param_id = VOCAB_ENDPOINT_URL + param_el["standard_name"]
         param = Parameter(
             id=param_el["name"],
-            observedProperty=ObservedProperty(id=param_el["name"], label=label),
+            observedProperty=ObservedProperty(id=param_id, label=label),
             type="Parameter",
-            unit=Unit(symbol=param_el["unit"]),
+            unit=Unit(symbol=symbol),
```

Comment:
So, for now we don't put a label in the `unit` (as there is in the example in #359).

Reply:
Ah, good point. I would like to follow #359. I think I forgot the label in the unit.

Reply:
Added the label.